extension/content_script.js

```diff
@@ -64,7 +63,9 @@ If a tracker is currently moving, stop it.
 See startMove()
 */
 function stopMove() {
-	if (timer) { 
+	if (timer) {
+		// Stop fading animation.
+		$("mark").stop();
```

Comment:
This is what allows partial fade.
When u lift key up, whatever fading animation is stopped there, indicating partial progress.

Reply:
Not true partial progress tho, cos when user resumes, we immediately go to next sentence

Reply:
Hmmmm yes, very cool to see if fading out. Definitely helps jarring nature of auto-read mode. Next step is probably getting "fade in" and also getting it to work for manual-read mode. 

LGTM

Reply:
Tracked in clickup - experiment w/ transition animation.